indexing-service/src/test/java/org/apache/druid/indexing/overlord/TaskQueueTest.java

```diff
@@ -519,6 +550,95 @@ public void testGetTaskStatus()
     Assert.assertEquals(TaskStatus.failure(failedTask, failedTask), taskQueue.getTaskStatus(failedTask).get());
   }
 
+  @Test
+  public void testGetActiveTaskRedactsPassword() throws JsonProcessingException
+  {
+    final String password = "AbCd_1234";
+    final ObjectMapper mapper = getObjectMapper();
+
+    final HttpInputSourceConfig httpInputSourceConfig = new HttpInputSourceConfig(Collections.singleton("http"));
```

Comment:
Nit: This need not be assigned to a variable, just construct it where needed.

Reply:
This is needed below to add a value to the injectable